airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/connectors/migrate_to_base_image/pipeline.py

```diff
@@ -342,10 +342,7 @@ async def run_connector_migration_to_base_image_pipeline(
                 steps_results.append(add_changelog_entry_result)
 
             # UPDATE DOC
-            add_build_instructions_to_doc = AddBuildInstructionsToReadme(
-                context,
-                add_changelog_entry_result.output,
-            )
+            add_build_instructions_to_doc = AddBuildInstructionsToReadme(context, steps_results[-1].output)
```

Comment:
Should this be under the `if pull_request_number is not None:` block?

Otherwise, `steps_results[-1]` will be `bump_version_in_metadata_result`.

Reply:
They actually had the same `output_directory` output, so it worked, but I updated the code to be more clear.

Reply:
👍 thanks.